extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -48,8 +48,17 @@ class AmpBaseCarousel extends PreactBaseElement {
   /** @override */
   init() {
     const {element} = this;
-    this.registerApiAction('prev', (api) => api.advance(-1), ActionTrust.LOW);
-    this.registerApiAction('next', (api) => api.advance(1), ActionTrust.LOW);
+    const advanceCount = Number(element.getAttribute('advance-count')) || 1;
+    this.registerApiAction(
+      'prev',
+      (api) => api.advance(-advanceCount),
```

Comment:
I think this should be a component logic. It's pretty easy for us to expose a couple more imperative methods on a component that apply the `advanceCount` prop.

Reply:
Good idea, done. :)